examples/measure.py

```diff
@@ -0,0 +1,129 @@
+import great3sims
+import argparse
+import numpy
+import pyfits
+import sys
+import os
+import time
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+import lsst.afw.detection as afwDetection
+import lsst.afw.geom as afwGeom
+import lsst.afw.math as afwMath
+import lsst.meas.algorithms
+import lsst.meas.base as measBase
+import lsst.meas.extensions.shapeHSM as shape
+from lsst.meas.algorithms import SourceDetectionTask
+
+def getIndex(name, cat):
+    index = 0
+    for item in cat.columns:
+        if name == item.name:
+            return index
+        index = index + 1
+    return -1
+
+parser = argparse.ArgumentParser()
+parser.add_argument("type", help="Name of galaxy type (control or real_galaxy")
+parser.add_argument("psfs", help="location of psfs file or psfs dir")
+args = parser.parse_args()
+hdus = pyfits.open("%s/ground/constant/epoch_catalog-000-0.fits"%args.type)
+cat = hdus[1]
+exp = afwImage.ExposureF("%s/ground/constant/image-000-0.fits"%args.type)
+
+# The psfs may be stored in a directory with names psfs_nnnn.fits
+# or in a multiple hdu fits files.  If it is a multiple hdu fits file,
+# the header of each psf should contain a "PSF_NO" to indicate the original
+# number of the psf before packaging.  If "PSF_NO" is not in the header,
+# the Psfs are assumed to be numbered 1, N where N is the number of psfs
+psf_dict = None
```

Comment:
It doesn't matter as much given that this isn't part of the mainline stack, but it'd still be a good idea to get in the habit of using `camelCase` names for variable names rather than using underscores here (and below).